xwiki-platform-core/xwiki-platform-alerts/xwiki-platform-alerts-ui/src/main/resources/XWiki/Alerts/Code/AlertsMenuUIX.xml

```diff
@@ -139,7 +139,7 @@
   ##
   #if ($stringtool.isNotBlank($menuContent.toString()))
     &lt;li class="dropdown" id="tmNotifications"&gt;
-      &lt;a class="icon-navbar dropdown-toggle" data-toggle="dropdown" role="button" title="$services.localization.render('watchlist')"&gt;&lt;span class="sr-only"&gt;$services.localization.render('core.menu.toggleNavigation')&lt;/span&gt;$services.icon.renderHTML('bell')&lt;/a&gt;
+      &lt;a class="icon-navbar dropdown-toggle" data-toggle="dropdown" role="button" title="$services.localization.render('watchlist.notifications')"&gt;&lt;span class="sr-only"&gt;$services.localization.render('core.menu.toggleNavigation')&lt;/span&gt;$services.icon.renderHTML('bell')&lt;/a&gt;
```

Comment:
It would be nice if you could provide some indentation here, so the line doesn't reach the 120 chars limit.